airflow_ext/gfw/models.py

```diff
@@ -79,7 +79,7 @@ def table_sensor(self, dag, task_id, project, dataset, table, date=None):
             table_id=self.format_date_sharded_table(table, date),
             poke_interval=10,  # check every 10 seconds for a minute
             timeout=60,
-            retries=24 * 7,  # retry once per hour for a week
+            retries=24,  # retry once per hour for a day
```

Comment:
@smpiano let's take the opportunity of doing some changes here.

1- Checking the documentation: https://airflow.apache.org/_modules/airflow/sensors/base_sensor_operator.html the base sensor has two modes, `poke` and `reschedule`. Based on the documentation, the `poke` is intended for sensors that have a short lifespan. I would use `mode=reschedule`.  The main difference is that the sensor will free the work slot.

2- Increase the poke interval to 10 minutes. We don't need to check every 10 seconds, that will overload the scheduler. So it will be `600` seconds.

3- Increase the timeout to 24hs = 60*60*24

4- Remove the retries parameter, retry_delay, retry_exponential_backoff